src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -836,6 +837,9 @@ let ``Shows help text for submit command`` () =
                       %s submit [options]
 
                     Options:
+                      -n <n> (REQUIRED)                                   A number.
+                      --pauli <PauliI|PauliX|PauliY|PauliZ> (REQUIRED)    The name of a Pauli matrix.
+                      --my-cool-bool (REQUIRED)                           A neat bit.
```

Comment:
:)

Reply:
😄 I don't think I can take credit for that. I believe that was Sarah originally; I just moved the entry point's arguments to the top of the list.